.github/workflows/darwin.yaml

```
@@ -128,6 +128,12 @@ jobs:
               timeout-minutes: 10
               run: |
                   scripts/examples/gn_build_example.sh examples/ota-provider-app/linux out/debug chip_config_network_layer_ble=false
+            - name: Build OTA image files with software version number 5
+              timeout-minutes: 10
+              run: |
+                  scripts/examples/gn_build_example.sh examples/ota-requestor-app/linux out/debug chip_config_network_layer_ble=false non_spec_compliant_ota_action_delay_floor=0 chip_device_config_device_software_version=5 chip_device_config_device_software_version_string="\"5.0\""
+                  src/app/ota_image_tool.py create -v 0xFF01 -p 0x8001 -vn 5 -vs "5.0" -da sha256 out/debug/chip-ota-requestor-app /tmp/test004-image
+                  src/app/ota_image_tool.py extract /tmp/test004-image /tmp/test004-raw-image
```

```suggestion
                  src/app/ota_image_tool.py extract /tmp/test005-image /tmp/test005-raw-image
```
That said, seems like `cp out/debug/chip-ota-requestor-app /tmp/test005-raw-image` would do the same thing, and might be clearer...